src/Exceptions/AccessDeniedException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace DeliciousBrains\SpinupWp\Exceptions;
+
+use Exception;
+
+class AccessDeniedException extends Exception
+{
+    public function __construct()
+    {
+        parent::__construct('Your API token does not have permission to that resource.', 401);
```

Comment:
@ohryan @danielmlozano should this not be `403`?

Reply:
Right, what the heck! Fixed.